modules/ossm-adding-project-using-smm-resource-console.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-mesh.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-smm-resource-console_{context}"]
+= Adding a project to the service mesh using the ServiceMeshMember resource with the web console
+
```

Comment:
Same comment as above. This proc module needs a brief introductory sentence or paragraph.

Reply:
Done